### DISCUSSION TITLE
How to add a field to an existing collection?

### DISCUSSION THREAD

**Discussion by zhanaali** [milvus-io/milvus#26601] [Q&A and General discussion]
How to add a field to an existing collection?

**Comment by xiaofan-luan** [comment]
Milvus don't support to add a new field. But you can setup a dynamic field (JSON) if you want

**Comment by xiaofan-luan** [comment]
we have the plan to do it on 2.4

**Reply by kent0113** [reply]
Does 2.4 support add a field to an existing collections?

**Reply by xiaofan-luan** [reply]
Not yet. due to lack of resources, we postpone this future  to 3.0
